tests/testthat/_snaps/update-news.md

```diff
@@ -78,9 +96,10 @@
       <simpleError in gh::gh(glue("GET /repos/{slug}/pulls/{pr_number}")): bla>
     Message
       ! Could not get title for PR #332
+      No encoding supplied: defaulting to UTF-8.
```

Comment:
This looks like we could fix it?

Reply:
Not really https://github.com/nealrichardson/httptest/issues/75

For another test I can use suppressMessages() but in this one we are recording our own message as well.